arangod/Cluster/Maintenance.cpp

```diff
@@ -447,10 +480,10 @@ arangodb::Result arangodb::maintenance::diffPlanLocal(
     auto const& dbname = pdb.key.copyString();
     if (!local.hasKey(dbname)) {
       if (errors.databases.find(dbname) == errors.databases.end()) {
-        actions.emplace_back(
-            ActionDescription({{std::string(NAME), std::string(CREATE_DATABASE)},
-                               {std::string(DATABASE), std::string(dbname)}},
-                              HIGHER_PRIORITY));
+        actions.emplace_back(ActionDescription(
+              std::map<std::string, std::string>{{std::string(NAME), std::string(CREATE_DATABASE)}, {std::string("tick"), std::to_string(TRI_NewTickServer())},
```

Comment:
The "tick" value here will always be unique and will not be equal to the database's id. What's its purpose?
I think it deserves a code comment.

Reply:
tick attribute removed